pkg/controllers/raycluster_controller.go

```diff
@@ -197,6 +201,25 @@ func (r *RayClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		}
 	}
 
+	// Locate the KubeRay operator deployment:
+	// - First try to get the ODH / RHOAI application namespace from the DSCInitialization
+	// - Or fallback to the well-known defaults
+	var kubeRayNamespaces []string
+	dsci := &dsciv1.DSCInitialization{}
+	err := r.Client.Get(ctx, client.ObjectKey{Name: "default-dsci"}, dsci)
```

Comment:
Can you load DSCI without hardcoding name? This will make code more resilient for possible naming changes.

Reply:
default-dsci is the one created by ODH/RHOAI operator.
99% case i would expect it is using this name, but ofc if user tried to create CR by themselves, that will do.
for the time being, we could hardcode with this name. 
a follow up PR can add more logic "check CR, get name, set here"